spec/lib/evm_database_ops_spec.rb

```diff
@@ -74,6 +87,15 @@
       allow(described_class).to receive(:backup_file_name).and_return("miq_backup")
       expect(PostgresAdmin).to receive(:backup).with(run_db_ops)
 
+      # This is now required after moving validate_free_space to inside the
+      # with_file_storage method, and wrapping it in a class check of:
+      #
+      #     if file_storage.class <= MiqGenericMountSession
+      #
+      # This "stubs the stub" to have it act like a MiqSmbSession, when in
+      # fact, it isn't.
+      allow(file_storage).to receive(:class).and_return(MiqSmbSession)
```

Comment:
Is there any reason this can't live in the definition of the double? For example, will it break other specs? It seems like a reasonable stub in any case.

Reply:
Since I am fixing the specs for this (without my patch):

Yes, this is necessary to be here.  It causes a bunch of other specs to fail if you do it globally.